src/Tizen.NUI.Components/Controls/SelectButton.cs

```diff
@@ -173,21 +178,16 @@ protected override bool HandleControlStateOnTouch(Touch touch)
                 return false;
             }
 
-            PointStateType state = touch.GetState(0);
-            bool ret = base.HandleControlStateOnTouch(touch);
-            switch (state)
+            if ((EnableToggleSelected == false) && (IsSelected == true))
```

Comment:
When the user touches the `selected` radio button, it should appear like this,

⊙ -> ● -> ⊙

To do that, the control state should be change like below when touching it.

1) Pressed & Selected (●)
2) Selected (⊙)

Since current NUI has a bug on Selector's priority, so you may not see the state Pressed&Selected visually, but after fixing it with #2122, you can see the Pressed&Selected state appears in your eyes.

The state changing on touch is handled in `Button.HandleControlStateOnTouch()`.
So if you return the code here, the radiobutton can not reach to the Pressed&Seleted state.

Reply:
Thank you for your comment.
I have updated the patch to support the transition when the RadioButton is pressed while it is selected.